clusters/testnet-eu/testnet/sealed-secret.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: mirror
+  namespace: tesnet
```

It's a bit presumptuous to name it after yourself, no? 😉 

```suggestion
  namespace: testnet
```

But as a result, it doesn't decrypt and I can't verify contents.

---

lol. rectified